app/View/Components/TranslatableInput.php

```diff
@@ -56,12 +56,26 @@ class TranslatableInput extends Component
      */
     public $required;
 
+    /**
+     * The name for asl or lsq content.
+     *
+     * @var bool
+     */
+    public $interpretationName;
+
+    /**
+     * The namespace for asl or lsq content.
+     *
+     * @var bool
+     */
+    public $interpretationNameSpace;
+
     /**
      * Create a new component instance.
      *
      * @return void
      */
-    public function __construct($name, $label, $hint = null, $model = null, $shortLabel = '', $required = false)
+    public function __construct($name, $label, $hint = null, $model = null, $shortLabel = '', $required = false, $interpretationName = '', $interpretationNameSpace = '')
```

Comment:
Should make the $interpretationName and $interpretationNameSpace default to `null`.